compel/include/uapi/ptrace.h

```diff
@@ -66,14 +66,14 @@ typedef struct {
 } seccomp_metadata_t;
 
 #ifndef PTRACE_GET_RSEQ_CONFIGURATION
-#define PTRACE_GET_RSEQ_CONFIGURATION	0x420f
-
-struct ptrace_rseq_configuration {
-	__u64 rseq_abi_pointer;
-	__u32 rseq_abi_size;
-	__u32 signature;
-	__u32 flags;
-	__u32 pad;
+#define PTRACE_GET_RSEQ_CONFIGURATION 0x420f
+
+struct __ptrace_rseq_configuration {
+	uint64_t rseq_abi_pointer;
+	uint32_t rseq_abi_size;
+	uint32_t signature;
+	uint32_t flags;
+	uint32_t pad;
```

Comment:
Would it be possible to move this change in the previous commit "cr-check: Add ptrace rseq conf dump feature", i.e., the commit introducing `ptrace_rseq_configuration`?